### DISCUSSION TITLE
How do I configure custom actions?

### DISCUSSION THREAD

**Discussion by eamonburns** [todotxt/todo.txt-cli#426] [Q&A] [answered]
When I type `todo.sh -h` at the bottom it says "Actions can be added and overridden using scripts in the actions directory."
- Where is this directory located?
- Can I change its location?
- Do I have to name the scripts a specific way? (e.g. `action` instead of `action.sh`)
- Is there official documentation that I can look at?

**Comment by inkarkat** [comment, marked as answer]
Custom add-ons (also named actions, although add-ons could also provide filters) are documented in the Wiki: https://github.com/todotxt/todo.txt-cli/wiki/Creating-and-Installing-Add-ons

A good place to start is by looking (or even using - why reinvent the wheel) at existing add-ons. Many are listed in the https://github.com/todotxt/todo.txt-cli/wiki/Todo.sh-Add-on-Directory
